superset/connectors/druid/models.py

```diff
@@ -1322,13 +1347,13 @@ def run_query(  # druid
                     set([x for x in pre_qry_dims if not isinstance(x, dict)])
                 )
                 dict_dims = [x for x in pre_qry_dims if isinstance(x, dict)]
-                pre_qry["dimensions"] = non_dict_dims + dict_dims
+                pre_qry["dimensions"] = non_dict_dims + dict_dims  # type: ignore
 
-                order_by = None
+                order_by = None  # type: ignore
```

Comment:
If we define this as `order_by: Optional[Metric]`, we can probably omit the ignore?

Reply:
@villebro the issue here is that `order_by` has already been defined on line 1291, 

```python
order_by = utils.get_metric_name(timeseries_limit_metric)
```

I declared the type as `order_by: Optional[str] = None`.